Wikipedia/UI-V5/WMFArticleContainerViewController.m

```diff
@@ -364,6 +355,9 @@ - (UIBarButtonItem*)shareToolbarItem {
 - (void)viewDidLoad {
     [super viewDidLoad];
 
+    [self setupToolbar];
```

Comment:
another option is to manually call `setToolbarHidden:animated:` from within this class, and still rely on the current logic to hide it when the next VC comes in if `toolbarItems` is empty